config/clusters/qcl/cluster.yaml

```diff
@@ -0,0 +1,24 @@
+name: qcl
+provider: gcp # https://console.cloud.google.com/kubernetes/clusters/details/europe-west1/qcl-cluster/observability?project=qcl-hub
+gcp:
+  key: enc-deployer-credentials.secret.json
+  project: qcl-hub
+  cluster: qcl-cluster
+  # We default to a regional cluster
+  zone: europe-west1
+support:
+  helm_chart_values_files:
+    - support.values.yaml
+    - enc-support.secret.values.yaml
+hubs:
+  - name: staging
+    # Tip: consider changing this to something more human friendly
+    display_name: "qcl - staging"
+    domain: staging.qcl.2i2c.cloud
```

Comment:
I think this should `staging.quantifiedcarbon.com` as they have provided their own domain name. I believe this will be the public URL that we request a https certificate for etc.

```
nslookup staging.quantifiedcarbon.com
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
staging.quantifiedcarbon.com	canonical name = staging.qcl.2i2c.cloud.
Name:	staging.qcl.2i2c.cloud
Address: 35.223.43.171
```

Reply:
```suggestion
    domain: staging.quantifiedcarbon.com
```